main.go

```diff
@@ -454,19 +455,21 @@ func watchBackend(cfg *config.Config, first chan bool) {
 		man := registry.Default.WatchManual()
 
 		for {
+
 			select {
 			case svccfg = <-svc:
 			case mancfg = <-man:
 			}
-
 			// manual config overrides service config
 			// order matters
-			next := svccfg + "\n" + mancfg
-			if next == last {
+			next.Reset()
+			next.WriteString(svccfg)
+			next.WriteString("\n")
+			next.WriteString(mancfg)
```

Comment:
Just call WriteString once with the original format? `svccfg + "\n" + mancfg`

Reply:
Yeah, I thought about that myself, but when a peer of mine reviewed it, he suggested to break it apart.  Would you expect any difference performance wise if it was all in one line?

Reply:
Very minimal ... you are potentially growing the buffer more than once.

https://golang.org/src/bytes/buffer.go?s=6142:6199#L173

Reply:
This example does show it broken out into multiple lines:
http://programming-tips.info/concatenate_string_literal/go/en/index.html

Reply:
fair enough, let's do it.

Reply:
I don't think that would be a big deal.  I'll update the PR.